Sources_v3/StreamChat/Query/Sorting/ChannelListSortingKey.swift

```diff
@@ -16,6 +16,9 @@ public enum ChannelListSortingKey: String, SortingKey {
     case lastMessageAt
     /// Sort channels by number of members.
     case memberCount
+    /// Sort channels by `cid`.
+    /// **Note**: Avoid using this one cause it can be expensive from backend side.
```

Comment:
This sound quite harsh :) If we don't want users to use it (there's probably no reason why they even should), let's not make it public.

Reply:
After discussions decided to go with updated comment.
See 34bee8a.